azure-fs-aks/Azure.Aks.fsproj

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Program.fs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Pulumi.Azure" Version="1.13.0-preview-alpha.1580890528" />
+    <PackageReference Include="Pulumi.FSharp" Version="1.11.0-preview-alpha.1581361451" />
+    <PackageReference Include="Pulumi.Random" Version="1.6.0-preview-alpha.1580429557" />
+    <PackageReference Include="Pulumi.Tls" Version="1.5.0-preview-alpha.1580429762" />
```

Comment:
If possible, please reference "stable" versions (ending with `-preview` with no extra suffix). The ones in your file are PR/nightly builds.

Reply:
Will fix.